.travis.yml

```diff
@@ -38,4 +38,6 @@ script:
   - cargo build
   - cargo build --no-default-features --features=portable
   - cargo test
+  - cargo run --bin test_secp256k1 --features="benchmarksecp256k1"
+  - cargo run --bin test_ed25519 --features="benchmarked25519"
```

Comment:
Why are these tests in `libursa/bin` and not somewhere in `libursa/src/signatures/`?

Reply:
These are binaries that run and are not libraries.

Reply:
Which binaries are being tested? The test appears to be calling library functions

Reply:
It's testing the speed of secp256k1 signatures and ed25519 signatures respectively. These tests can be run against portable or default mode.